contribs/gmf/apps/desktop/default.js

```diff
@@ -1,12 +1,21 @@
+var urlElements = window.location.pathname.split('/');
+
+var gmfUrlElements = urlElements.slice(0, urlElements.length - 3);
+gmfUrlElements.push('src', 'directives', 'partials')
```

Comment:
Are these files not scanned by eslint? The missing `var ...` was also not detected.